src/Middleware/CanAtLeastMiddleware.php

```diff
@@ -14,9 +14,9 @@ class CanAtLeastMiddleware
      * @param  string $permissions
      * @return mixed
      */
-    public function handle($request, Closure $next, $permissions)
+    public function handle($request, Closure $next, ...$permissions)
```

Comment:
Yes, definitively the `$permissions` is always an array, so the variadic it's not necessary.